THEME.md

```diff
@@ -209,21 +209,36 @@ The following will inject the action link (or button) into the e-mail:
 
 ```html
 <!-- Action -->
+<% if (locals.action.constructor !== Array) { %>
```

Comment:
@collnwalkr Oops, my bad, that was an old comment that got submitted for some reason.